ansible/reset_environment.yml

```diff
@@ -20,6 +20,26 @@
     - name: Setting fact of security group id
       set_fact:
         my_security_group_id: "{{ hostvars.localhost.my_ec2_facts.instances[0].groups[0].id }}"
+    - name: Override variables if 'deploy_helios' is 'true'
```

Comment:
What do you think about moving this block that is repeated in all three files to it's own file, then we include that single file in each of the reset_environment/setup_environment/setup_local_environment..?

Trying to see if we can avoid duplicating the same code in 3 chunks.